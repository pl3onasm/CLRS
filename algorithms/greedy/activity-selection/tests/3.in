[151, 157), [531, 535), [114, 118), [186, 191), [285, 290), [80, 85), [409, 413), [201, 207), [44, 50), [569, 574), [290, 294), [152, 157), [1, 6), [401, 407), [564, 568), [15, 19), [254, 258), [25, 30), [319, 323), [551, 557), [185, 190), [109, 114), [6, 12), [79, 83), [312, 317), [90, 95), [302, 307), [35, 40), [112, 118), [324, 328), [57, 62), [379, 385), [150, 156), [31, 36), [140, 145), [259, 265), [58, 62), [50, 55), [13, 18), [144, 149), [76, 81), [18, 23), [58, 62), [379, 384), [141, 147), [430, 434), [467, 472), [267, 273), [74, 78), [512, 517), [350, 355), [86, 91), [398, 402), [503, 508), [413, 419), [178, 183), [70, 74), [345, 350), [237, 242), [128, 134), [463, 469), [18, 24), [450, 455), [278, 283), [503, 509), [124, 128), [24, 28), [97, 101), [172, 178), [438, 443), [523, 529), [244, 248), [346, 352), [207, 213), [295, 301), [213, 219), [385, 391), [65, 70), [504, 509), [113, 119), [310, 314), [275, 280), [355, 361), [217, 222), [373, 379), [156, 161), [476, 481), [129, 134), [476, 482), [417, 423), [503, 508), [481, 486), [54, 59), [307, 313), [222, 227), [134, 139), [431, 436), [191, 196), [28, 32), [10, 15)
