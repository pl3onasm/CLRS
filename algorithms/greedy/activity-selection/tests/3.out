Maximum number of activities: 67

Activity sequence:

Activity 13: [1, 6)
Activity 23: [6, 12)
Activity 39: [13, 18)
Activity 42: [18, 23)
Activity 67: [24, 28)
Activity 99: [28, 32)
Activity 28: [35, 40)
Activity 9: [44, 50)
Activity 38: [50, 55)
Activity 43: [58, 62)
Activity 78: [65, 70)
Activity 57: [70, 74)
Activity 49: [74, 78)
Activity 24: [79, 83)
Activity 52: [86, 91)
Activity 68: [97, 101)
Activity 22: [109, 114)
Activity 3: [114, 118)
Activity 66: [124, 128)
Activity 88: [129, 134)
Activity 96: [134, 139)
Activity 35: [140, 145)
Activity 33: [150, 156)
Activity 86: [156, 161)
Activity 69: [172, 178)
Activity 56: [178, 183)
Activity 21: [185, 190)
Activity 98: [191, 196)
Activity 8: [201, 207)
Activity 74: [207, 213)
Activity 76: [213, 219)
Activity 95: [222, 227)
Activity 59: [237, 242)
Activity 72: [244, 248)
Activity 17: [254, 258)
Activity 36: [259, 265)
Activity 48: [267, 273)
Activity 82: [275, 280)
Activity 5: [285, 290)
Activity 11: [290, 294)
Activity 75: [295, 301)
Activity 27: [302, 307)
Activity 94: [307, 313)
Activity 19: [319, 323)
Activity 30: [324, 328)
Activity 58: [345, 350)
Activity 51: [350, 355)
Activity 83: [355, 361)
Activity 85: [373, 379)
Activity 44: [379, 384)
Activity 77: [385, 391)
Activity 53: [398, 402)
Activity 7: [409, 413)
Activity 55: [413, 419)
Activity 46: [430, 434)
Activity 70: [438, 443)
Activity 63: [450, 455)
Activity 61: [463, 469)
Activity 87: [476, 481)
Activity 92: [481, 486)
Activity 91: [503, 508)
Activity 50: [512, 517)
Activity 71: [523, 529)
Activity 2: [531, 535)
Activity 20: [551, 557)
Activity 15: [564, 568)
Activity 10: [569, 574)
